### PR TITLE
Password string

### PR DESCRIPTION
**Pull request by ericawright**
Fixes: #621.
Fixes: #622.

### CODE REVIEW

File: public/locales/en-US/send.ftl [hunk 1]
```diff
@@ -87,3 +87,4 @@ footerLinkCookies = Cookies
 requirePasswordCheckbox = Require a password to download this file
 addPasswordButton = Add password
 passwordTryAgain = Incorrect password. Try again.
+passwordResult = Password:
```
Comment (flodolo) [inline public/locales/en-US/send.ftl]: Can you add a short localization note? e.g.

```
// This label is followed by the password set to download a file
```

Is it just me or #622 shows a completely different string?

Comment (ericawright) [reply public/locales/en-US/send.ftl]: It does, though I believe the consensus was to change it to just be "password", though it's possible that's just what it became in the last iteration....

Comment (ehuggett) [reply public/locales/en-US/send.ftl]: https://github.com/mozilla/send/issues/381#issuecomment-327930546 has the mockups (last image in "sender" shows the string "Password required to download this file:")

Comment (ericawright) [reply public/locales/en-US/send.ftl]: I double-checked with some folks, and the long one is preferred. Changing it, thanks!

**Comment by dannycoates** [comment]
This looks good, but I made some other changes in another PR, so I'm closing this...

To maybe make RTL better, in #632, I made the password an ftl variable, and also some inside baseball changes to make other weird passwords (besides multiple spaces) work better.

**Pull request closed without merging**